Task 3.py

```diff
@@ -0,0 +1,11 @@
+def reverse(line):
+    reversedline = ''
+    for i in line:
+        reversedline = i + reversedline
+    return reversedline
+
+
+usernumber = input('Пожалуйста, введите число: ')
+newnumber = reverse(usernumber)
+
+print('Old number:', usernumber, 'New number:', newnumber)
```

Comment:
если ввести 1000 то ответ не верный
```suggestion
print('Old number:', usernumber, 'New number:', int(newnumber))
```